app/views/practices/_practice_editor_sidenav.html.erb

```diff
@@ -12,7 +12,11 @@
           <% if current_page?(practice_introduction_path(@practice)) %>
             <a href="#" onclick="return false" class="width-full side-nav-highlighted"><%= @practice.name %></a>
           <% else %>
-            <%= link_to @practice.name, practice_introduction_path(@practice), id: "sideNavPracticeNameLink", class: "dm-tab #{'side-nav-highlighted' if !current_page?(practice_instructions_path(@practice)) && !current_page?(practice_metrics_path(@practice))}" %>
+            <% if !PracticeEditorSession.practice_locked_for_current_user(current_user[:id], @practice.id) %>
+              <a id="sideNavPracticeNameLinkStartTimer" href="<%=practice_introduction_path(@practice)%>" class="dm-tab <% if !current_page?(practice_instructions_path(@practice)) && !current_page?(practice_metrics_path(@practice)) %>side-nav-highlighted<% end %>"><%= @practice.name %></a>
```

Comment:
I think we can use a `link_to` method here as well, just to keep it congruent with the one below.

Reply:
Yep i switched it to use link_to